python/sglang/srt/_custom_ops.py

```diff
@@ -77,42 +80,89 @@ def init_custom_ar(
             rank: int,
             full_nvlink: bool,
         ) -> int:
-            return sgl_kernel.allreduce.init_custom_ar(
-                meta, rank_data, handles, offsets, rank, full_nvlink
+            init_func = (
+                aiter_custom_ar.init_custom_ar
+                if get_bool_env_var("CK_MOE")
```

Comment:
get_bool_env_var("CK_MOE") must be used together with **is_hip**, would it better to provide hip env function ?

```suggestion
if get_hip_bool_env_var("CK_MOE")
```